fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, fapd_manager):
         UIConnectedWidget.__init__(self, get_system_feature())
         self.__events__ = [
             "analyze_button_pushed",
+            "refresh_toolbar",
```

Comment:
Maybe at some point, "refresh_toolbar" should be converted to something on the order of an event, e.g. "analysis complete" which might map into a redux state var if needed.